@here/harp-mapview/lib/MapView.ts

```diff
@@ -911,6 +916,32 @@ export class MapView extends THREE.EventDispatcher {
         this.m_theme.clearColor = theme.clearColor;
         this.renderer.setClearColor(new THREE.Color(theme.clearColor));
 
+        const effects = theme.effects;
+        if (effects !== undefined) {
+            if (effects.toneMappingExposure !== undefined) {
```

Comment:
Todo: also handle individual effects being undefined, if effects are not undefined.